crates/subspace-solving/src/lib.rs

```diff
@@ -174,3 +178,22 @@ pub fn derive_piece_chunk_otp(
         hash[0], hash[1], hash[2], hash[3], hash[4], hash[5], hash[6], hash[7],
     ]
 }
+
+/// Expand chunk to be the same size as solution range for further comparison
+pub fn expand_chunk(chunk: &BitSlice<u8, Lsb0>) -> SolutionRange {
+    let mut bytes = 0u64.to_le_bytes();
+
+    bytes
+        .view_bits_mut::<Lsb0>()
+        .iter_mut()
+        .zip(chunk)
+        .for_each(|(mut expanded, source)| {
+            *expanded = *source;
+        });
+
+    let hash = blake2b_256_hash(&bytes);
+
+    SolutionRange::from_le_bytes([
+        hash[0], hash[1], hash[2], hash[3], hash[4], hash[5], hash[6], hash[7],
+    ])
```

Comment:
I wanted to avoid `try_into()` and `unwrap()` by listing bytes explicitly. Is it that bad?

Reply:
Nope, I generally find it more readable. It is clear anyway that `unwrap` won't panic here

Reply:
We don't use `.unwrap()` in blockchain code, so it'll be `.expect()` anyway. Let's keep it as is for now if you don't mind, this may actually change soon anyway.